assets/sass/vendor/_mdc-dialog.scss

```diff
@@ -2,24 +2,17 @@
 .googlesitekit-plugin .mdc-dialog {
 	z-index: 10000;
 
-	%label-small {
-		font-size: $fs-body-sm;
-		letter-spacing: 0.2px;
-		line-height: $lh-label-sm;
-	}
-
 	.mdc-dialog__surface {
-		border-radius: 8px;
-		box-shadow: 0 10px 40px 0 rgba(#000, 0.35);
-		padding: $grid-gap-desktop + 12px $grid-gap-desktop * 2 - 10; // 36px 38px.
+		border-radius: $br-xs;
+		box-shadow: 0 10px 40px 0 rgba($c-black, 0.35);
+		max-width: 76vw;
+		padding: $grid-gap-desktop;
 
 		@media (min-width: $bp-tablet) {
-			max-width: 478px;
-		}
-
-		@media (max-width: $bp-mobileOnly) {
-			max-width: 76vw;
-			padding: $grid-gap-desktop;
+			/* Differs from what is seen in figma design - 478px,
+			after disscusion with Sigal https://10up.slack.com/archives/C01CA828ZPZ/p1707129589297719 */
```

Comment:
Let's remove this comment, as we do not reference Slack links in the codebase. In similar cases, a simple comment in the GitHub PR for the code reviewer is sufficient, for example, see [this](https://github.com/google/site-kit-wp/pull/8221#discussion_r1477171502).

```suggestion
```